### ISSUE TITLE
VAMC real-time banners tech debt cleanup: remove flipper / feature toggle logic and flippers/ toggles

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#19468]
If we have created flippers for `banner_use_alternative_banners` in vets-api or CMS feature toggles, we need to: 

## Acceptance criteria
- [ ] Update the logic in FE templates to no longer be conditional / use the toggles
- [ ] Remove CMS feature toggle: `feature_banner_use_alternative_banners`
- [ ] Remove vets-api flipper: `banner_use_alternative_banners`
    - [ ] Ask Platform support to run the sidekiq script that updates prod flippers, and will remove the flipper from the UI after our code change, e.g. https://dsva.slack.com/archives/CBU0KDSB1/p1730397715213859
- [ ] Remove context from Banners if no longer being utilized now that path is parsed https://github.com/department-of-veterans-affairs/vets-api/pull/19897

**Comment by Agile6MSkinner** [comment]
Calling this refined, but @SnowboardTechie needs to get the Flipper/Toggle names into the ticket. If they're not here yet, go after him.

**Comment by SnowboardTechie** [comment]
https://github.com/department-of-veterans-affairs/vets-api/pull/18913/files flipper being added

**Comment by jilladams** [comment]
WHen we pick this up, double check with Bryan / Daniel that the listed flipper names haven't changed - should be easy to tell in prod:  
* CMS feature toggle: `feature_banner_use_alternative_banners` - https://prod.cms.va.gov/admin/config/system/feature_toggle
* vets-api flipper: `banner_use_alternative_banners` - https://api.va.gov/flipper/features

**Comment by jilladams** [comment]
We decided today that based on Architecture intent for real-time banners, we need to keep both the CMS feature toggle and the vets-api flipper long-term, so we can cut back to content-build version of banners in the future as needed. I've rolled a documentation update about that into #11591 for the flipper. 

This ticket will close as no-op. 

However, one question first: @SnowboardTechie does this AC need to do anything different in light of the rest of this comment? "Remove context from Banners if no longer being utilized now that path is parsed https://github.com/department-of-veterans-affairs/vets-api/pull/19897"
